components/Footer/index.tsx

```diff
@@ -38,6 +38,12 @@ const Footer = () => (
                 </li>
               ))}
             </ul>
+            {meta.html ?
+            <div 
+              className="govuk-footer__meta-custom"
+              dangerouslySetInnerHTML={{__html: meta.html
```

Comment:
Sad times to be using `dangerouslySetInnerHTML` :( Any reason why we wouldn't add that paragraph straight in here?

Reply:
add as in hardcode?

Reply:
I suppose.

Reply:
ok, doing as we speak

Reply:
done